src/enums/ruleIds.ts

```diff
@@ -1,5 +1,5 @@
 /** Unique values to identify rules. */
-export enum Rules {
+export enum RuleIds {
```

Comment:
Renamed to reduce the number of things called "Rules". And also, this isn't an actual rule but an id used to identify a rule.

(There remains an interface named Rules... I'm not going to worry about that for now.)